src/browser/of_events.ts

```diff
@@ -39,9 +41,36 @@ class OFEvents extends EventEmitter {
                 // Wildcard on any channel/topic of a specified source (ex: 'window/*/myUUID-myWindow')
                 super.emit(route(channel, '*', source), envelope);
             }
+            if (channel === 'window') {
+                const propTopic = `window-${topic}`;
+                const dontPropagateToSystem = [
+                    'auth-requested'
+                ];
+                eventPropagations.set(route.application(propTopic, uuid), {...payload, type: propTopic, topic: 'application'});
```

Comment:
If we are going to be using the new (stage 4) spread syntax for an object literal and we are spreading on the `payload` variable, it should be declared as `object` instead of `any` because it is unlikely that we are intending to spread primitive types.

Reply:
In this case should we keep the payload, ...extraArgs signature?

Reply:
Not sure I see the relationship between where we declare `payload` (signature vs scoped const) and the type we declare it as. Pretty sure you can type declare `[object, any[]]` on the destructuring assignment.